### ISSUE TITLE
can't find saved weights after training

### ISSUE THREAD

**Issue by moyid** [HHousen/TransformerSum#33]
my model finished training but I can't seem to find the saved weights after it's done. I created a directory ./trained_models, but nothing is there. Here is my command:
`python src/main.py --model_name_or_path allenai/longformer-base-4096 --model_type longformer --data_path ./datasets/cnn_dm_extractive_compressed_small/ --weights_save_path ./trained_models --do_train --max_epochs 1 --no_use_token_type_ids --max_seq_length 2048 --batch_size 4 --log WARNING`

**Comment by salman1993** [comment]
It should be under `trained_models/lightning_logs/version_[VERSION_NUM]/checkpoints/epoch=[X].ckpt`

**Comment by moyid** [comment]
there's nothing in trained_models.

**Comment by moyid** [comment]
I think this had worked in the past with a different model, but it is not working anymore

**Comment by salman1993** [comment]
Yeah I can't see the directory now either but I see this `trained_models.ckpt` file in the directory I ran it from. 

It contains the model saved state and other config stuff, although I used this flag `--use_custom_checkpoint_callback` during training so that might be the case as well

```
In [3]: t = torch.load("trained_models.ckpt")

In [4]: t.keys()
Out[4]: dict_keys(['epoch', 'global_step', 'pytorch-lightning_version', 'callbacks', 'optimizer_states', 'lr_schedulers', 'state_dict', 'hparams_name', 'hyper_parameters'])

In [5]: t['epoch']
Out[5]: 1

In [6]: t['callbacks']
Out[6]:
{pytorch_lightning.callbacks.model_checkpoint.ModelCheckpoint: {'best_model_score': tensor(0.3369, device='cuda:0'),
  'best_model_path': '/home/salman/TransformerSum/trained_models.ckpt'}}
```

**Comment by moyid** [comment]
ok let me try again with that flag

**Comment by moyid** [comment]
I have made this work by setting use_logger tensorboard -- otherwise the checkpoint is saved in the wandb folder -- even though there doesn't appear to be tensorboard logging support. Also, I needed to set --use_custom_checkpoint_callback to get a checkpoint at the end of every epoch -- otherwise it only saves 1 checkpoint, and based on the label it is epoch0 -- if there's only 1 checkpoint I'd rather it be the last epoch.

**Comment by HHousen** [comment]
@moyid Try setting `--no_wandb_logger_log_model` and not setting the `--weights_save_path`.
